schema/deploy/warehouse/presence_absence.sql

```diff
@@ -0,0 +1,33 @@
+-- Deploy seattleflu/schema:warehouse/presence_absence to pg
+-- requires: warehouse/presence_absence
+-- requires: warehouse/target
+
+begin;
+
+set search_path to warehouse;
+
+create table presence_absence (
+    presence_absence_id integer primary key generated by default as identity,
+    identifier text unique not null,
+    sample_id integer references sample (sample_id) not null, 
+    target_id integer references target (target_id) not null,
+    present boolean,
+    well_details jsonb
+);
+
+-- We also opted for `sample_id` over `aliquot_id` as we are not sure if we will be receiving aliquot IDs
+--   in the presence-absence data. If we are, we will likely construct an aliquot table that keeps track of
+--   the sample_id, and reference only the aliquot_id in this table.
+
+comment on table presence_absence is 'A presence_absence test result from an individual sample';
+comment on column presence_absence.presence_absence_id is 'Internal id of this presence_absence test';
+comment on column presence_absence.identifier is 'External id of this presence_absence test';
+comment on column presence_absence.sample_id is 'The sample that is being tested for presence_absence of a target';
+comment on column presence_absence.target_id is 'The target for the presence_absence test';
+comment on column presence_absence.present is 'The result of a presence_absence test. True if the target is found in the sample';
+comment on column presence_absence.well_details is 'The details for each well of the presence absence test';
```

Comment:
All of these comments should be adjusted to use the term "presence/absence" instead of the SQL-safe `presence_absence`.